nrm_django/nrm_site/templates/admin/base.html

```diff
@@ -11,7 +11,7 @@
     <script src="{% static 'admin/js/nav_sidebar.js' %}" defer></script>
     {% endif %}
     <link href="{% static 'vendor/uswds-2.10.1/css/uswds.min.css' %}" rel="stylesheet" type="text/css" />
-    <link href="{% static 'css/site.css' %}" rel="stylesheet" type="text/css" />
+    <link href="{% static 'css/site.css' %}?v=8" rel="stylesheet" type="text/css" />
```

Comment:
Is this used for cache-busting? If so, do we want to change to a hash-based query parameter here?

Reply:
It was. We don't _really_ need to keep it at all.